### PR TITLE
Feat/add clickable row

### PR DESCRIPTION
**Pull request by meghaniankov**
# Description

Add ability to make entire row clickable. Added RowWrapper that adds link if clickableRow link is included in the schema.

# Checklist

Pull request contains:

- [ ] A new component
- [x] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [x] Includes theme changes for both Uswitch and money
- [x] Work has been tested in multiple browsers
- [x] PR description includes description of change
- [ ] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [x] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

File: src/compounds/product-table/src/components/rowWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      sx={{
+        textDecoration: 'none',
+        color: 'inherit',
+        variant: 'compounds.product-table.row.linkWrapper'
+      }}
+      href={link}
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      <div
```
Comment (jonathan-fielding) [inline src/compounds/product-table/src/components/rowWrapper.tsx:23]: could you put this padding up onto the linkWrapper?

Comment (meghaniankov) [reply src/compounds/product-table/src/components/rowWrapper.tsx:23]: If I move it into the anchor, it doesn't apply the padding correctly 
image

Comment (jonathan-fielding) [reply src/compounds/product-table/src/components/rowWrapper.tsx:23]: you probably need to add display: block to the anchor, anchors by default are inline so cant have padding

Comment (meghaniankov) [reply src/compounds/product-table/src/components/rowWrapper.tsx]: of course! just made the change

**Review by jonathan-fielding — DISMISSED** [review]

**Review by Segger87 — APPROVED** [review]

**Pull request merged**